integration_tests/src/tests/commit_post_test.rs

```diff
@@ -351,6 +354,12 @@ pub fn overdue_precommit_test(v: &dyn VM) {
                             // The call to burnt funds indicates the overdue precommit has been penalized
                             Expect::burn(id_addr.id().unwrap(), Some(precommit.pre_commit_deposit)),
                             // No re-enrollment of cron because burning of PCD discontinues miner cron scheduling
+                            Expect::power_update_pledge(
+                                id_addr.id().unwrap(),
+                                Some(TokenAmount::from_atto(BigInt::from_signed_bytes_be(&[
+                                    255, 17, 91, 207, 234, 13, 8, 99, 78,
```

Comment:
This is completely opaque to me. Please use de/serialization code to produce this value so we can see where it comes from.